lib/util/hud_lists_2024.rb

```diff
@@ -0,0 +1,1735 @@
+      3 => 'Child care',
+      4 => 'Criminal justice/legal services',
+      5 => 'Education',
+      6 => 'Employment and training services',
+      7 => 'Food/meals/nutritional services',
+      8 => 'Health/medical care',
+      9 => 'Life skills training',
+      10 => 'Mental health care/counseling',
+      11 => 'Outreach and/or engagement',
+      12 => 'Substance use services/treatment',
+      13 => 'Transportation',
+      14 => 'Other HOPWA funded service',
+    }.freeze
+  end
+
+  # W3
+  def no_assistance_reason_map
+    {
+      1 => 'Applied; decision pending',
+      2 => 'Applied; client not eligible',
+      3 => 'Client did not apply',
+      4 => 'Insurance type not applicable for this client',
+      8 => "Client doesn't know",
+      9 => 'Client prefers not to answer',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  # W4.3
+  def viral_load_available_map
+    {
+      0 => 'Not available',
+      1 => 'Available',
+      2 => 'Undetectable',
+      8 => "Client doesn't know",
+      9 => 'Client prefers not to answer',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  # W4.B
+  def t_cell_source_viral_load_source_map
+    {
+      1 => 'Medical Report',
+      2 => 'Client Report',
+      3 => 'Other',
+    }.freeze
+  end
+
+  # W5.1
+  def housing_assessment_at_exit_map
+    {
+      1 => 'Able to maintain the housing they had at project entry',
+      2 => 'Moved to new housing unit',
+      3 => 'Moved in with family/friends on a temporary basis',
+      4 => 'Moved in with family/friends on a permanent basis',
+      5 => 'Moved to a transitional or temporary housing facility or program',
+      6 => 'Client became homeless - moving to a shelter or other place unfit for human habitation',
+      7 => 'Jail/prison',
+      8 => "Client doesn't know",
+      9 => 'Client prefers not to answer',
+      10 => 'Deceased',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  # W5.A
+  def subsidy_information_map
+    {
+      1 => 'Without a subsidy 1',
+      2 => 'With the subsidy they had at project entry 1',
+      3 => 'With an on-going subsidy acquired since project entry 1',
+      4 => 'But only with other financial assistance 1',
+      11 => 'With on-going subsidy 2',
+      12 => 'Without an on-going subsidy 2',
+    }.freeze
+  end
+
+  # ad_hoc_yes_no
+  def ad_hoc_yes_no_map
+    {
+      0 => 'No',
+      1 => 'Yes',
+      8 => "Don't Know",
+      9 => 'Refused',
+      99 => 'Data not collected',
+    }.freeze
+  end
+
+  # race
+  def race_map
+    {
+      'AmIndAKNative' => 'American Indian, Alaska Native, or Indigenous',
+      'Asian' => 'Asian or Asian American',
+      'BlackAfAmerican' => 'Black, African American, or African',
+      'NativeHIPacific' => 'Native Hawaiian or Pacific Islander',
+      'White' => 'White',
+      'RaceNone' => "Doesn't Know, refused, or not collected",
+    }.freeze
+  end
+
+  def hud_code_to_function_map
```

Comment:
@eanders this thing makes me a little nervous. I was going through I noticed the old file had some inconsistency about whether the # was the data element ID or the list ID. HUD can also be inconsistent with this it seems.  It is used by this function https://github.com/greenriver/hmis-warehouse/blob/0ca9327e52da871eded2de8ccb460bfbcb16ddcc/lib/util/hud_utility.rb#L118-L126 

Search the codebase for `HudUtility.list` it's only used in 3 places, with hard coded ids, so those could easily be replaced with references to the maps directly. WOuld you be OK with making that change and removing this code mapping?